azure-devops/azext_devops/dev/team/feedback.py

```diff
@@ -14,4 +13,3 @@ def feedback():
     url = 'https://aka.ms/azure-devops-cli-feedback'
     print('Thank you for taking the time to share your feedback. Please submit your feedback on the following web ' +
           'page: {url}'.format(url=url))
-    open_url(url=url)
```

Comment:
not needed actually :) but no harm in removing as well

Reply:
Reason for removing: Since it auto opens the number of hits to the URL will be high. By doing this I am hoping the hits will come down significantly and then we can safely remove the command.